app/src/main/java/org/thoughtcrime/securesms/conversation/v2/messages/VisibleMessageView.kt

```diff
@@ -123,7 +127,14 @@ class VisibleMessageView : LinearLayout {
             binding.profilePictureView.glide = glide
             binding.profilePictureView.update(message.individualRecipient)
             binding.profilePictureView.setOnClickListener {
-                showUserDetails(senderSessionID, threadID)
+                if (thread.isOpenGroupRecipient && IdPrefix.fromValue(senderSessionID) == IdPrefix.BLINDED) {
```

Comment:
Should we move this check into the `UserDetailsBottomSheet` so the user isn't just presented with a new activity for blinded IDs?

Reply:
Yes, will remove this